app/javascript/suggestions.js

```diff
@@ -102,7 +102,6 @@ Input.prototype.updateSuggestionsWithOptions = function (options) {
     li.textContent = option.textContent
     li.setAttribute('role', 'option')
     li.setAttribute('tabindex', '-1')
-    li.setAttribute('aria-selected', option.value === this.$module.value)
```

Comment:
I wonder if we should comment this out instead with a comment. We now have 3 versions of this code, and it might be good to communicate which changes we have made to the original